src/swap/defi/thorchain/swapkit.ts

```diff
@@ -92,6 +89,9 @@ const asThorSwapRoute = asObject({
   // expectedOutputMaxSlippage: asString,
   // expectedOutputUSD: asString,
   // expectedOutputMaxSlippageUSD: asString,
+  transaction: asObject({
```

Comment:
Are you for sure sure that this isn't optional. There may not be a data payload for some types of swaps? Just make sure to test that this is true when sending EVM, UTXO, and Solana (if supported)